src/compiler/parser.ts

```diff
@@ -436,6 +436,13 @@ module ts {
         // attached to the EOF token.
         let parseErrorBeforeNextFinishedNode: boolean = false;
 
+        export const enum StatementFlags {

```

Comment:
It takes StatementFlags because we need them to test against the StatementFlags returned by getDeclarationFlags within the function.